insights/parsers/du_usage.py

```diff
@@ -0,0 +1,60 @@
+"""
+du - Command
+============
+
+Module for the processing of output from the ``du`` command stored in
+file.  The base class ``DBDiskSpace`` provides all of the functionality.
+
+This example demonstrates the funcionality of ``DBDiskSpace`` class.
+
+Examples:
+    >>> FILE_CONTENT = '''
+    ... COMMAND> du -sh /var/lib/mongodb
+    ...
+    ... 22G     /var/lib/mongodb
+    ... 635M    /var/lib/pgsql
```

Comment:
As `du -sh` command particularly asks for `/var/log/mongodb`, the output should contain usage for only that path.

Reply:
updated this.